package/kedro_viz/integrations/kedro/telemetry.py

```diff
@@ -1,14 +1,15 @@
 """`kedro_viz.integrations.kedro.telemetry` helps integrate Kedro-Viz with Kedro-Telemetry
 """
-import hashlib
-import socket
+
 from pathlib import Path
 from typing import Optional
 
-import yaml
-
 try:
-    from kedro_telemetry.plugin import _get_heap_app_id, _is_valid_syntax
+    from kedro_telemetry.plugin import (
+        _check_for_telemetry_consent,
+        _get_heap_app_id,
+        _get_or_create_uuid,
+    )
 
```

Comment:
does this mean, next version of viz should have the latest kedro telemetry to be installed for tracking or is there a way to get latest viz support older kedro telemetry ?